package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "profiler-server",
+  "version": "1.0.0",
+  "description": "This is the server-side part of the Firefox Profiler.",
+  "main": "index.js",
+  "repository": "https://github.com/firefox-devtools/profiler",
+  "author": "Mozilla",
+  "license": "MPL2",
+  "private": true,
+  "scripts": {
+    "flow": "flow --max-warnings 0",
+    "flow:ci": "flow check --max-warnings 0",
+    "flow-stop": "flow stop",
+    "flow-coverage": "flow-coverage-report -i '**/*.js' -t html -t text",
+    "flow-generate-libdefs": "flow-typed install --libdefDir src/types/libdef",
+    "lint": "eslint .",
+    "lint-fix": "yarn lint-js --fix",
+    "preinstall": "node bin/pre-install.js",
+    "build:clean": "rimraf dist && mkdirp dist",
+    "build": "babel src --out-dir dist",
+    "serve": "node dist/index.js",
+    "start": "nodemon --exec babel-node -w src src/index.js",
```

Comment:
I tried for some time to make it work without babel-node but this is more reliable like this.